src/DatePickerInput.js

```diff
@@ -215,10 +209,10 @@ const DatePickerInput = React.createClass({
     return (
       <div
         className={cx('react-datepicker-component', className)}
-        style={style}
-        onClick={this.stopPropagation}>
+        style={style}>
         <div className='react-datepicker-input'>
           <input
+            ref='datePickerInput'
```

Comment:
When talking of `DatePicker` I was referring to this part: https://github.com/buildo/rc-datepicker/blob/master/src/DatePicker.js#L67-L71
I think we should remove that too (It was a naive hack to avoid closing it when clicking inside)